Description: Upstream changes introduced in version 0.1.1~bzr97-0~10.04~dockers1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 dockmanager (0.1.1~bzr97-0~10.04~dockers1) lucid; urgency=low
 .
   * New development release of lp:dockmanager bzr97
 .
 The person named in the Author field signed this changelog entry.
Author: Rico Tzschichholz <ricotz@ubuntu.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ dockmanager-0.1.1~bzr97/.version_info
@@ -0,0 +1 @@
+bzr dockmanager r97 ppa
\ No newline at end of file
